src/app/components/page-alert/page-alert.component.ts

```diff
@@ -25,10 +27,10 @@ export class PageAlertComponent extends BaseComponent implements OnInit {
 
     public ngOnInit(): void {
         this.store.takeUntil(this.ngUnsubscribe)
-            .pipe(select('rubrics'))
-            .subscribe((state: RubricsStateInterface) => {
-                if (state.error !== undefined) {
-                    this.showError(state.error.message);
+            .pipe(select(selectError))
+            .subscribe((error: DisplayableErrorInterface | undefined) => {
+                if (error !== undefined) {
+                    this.showError(error.message);
```

Comment:
@hollielu what do you think about refactoring this to use a `filter` operator instead of the `if` block? Good practice for learning rxjs at least.